scw/request.go

```diff
@@ -34,11 +34,12 @@ func (req *ScalewayRequest) getAllHeaders(token auth.Auth, userAgent string, ano
 
 	allHeaders.Set("User-Agent", userAgent)
 	if req.Body != nil {
-		allHeaders.Set("content-type", "application/json")
```

Comment:
FYI: you don't need to change this line because the key is formatted https://golang.org/src/net/textproto/reader.go?s=15268:15312#L563